manala.influxdb/defaults/main.yml

```diff
@@ -5,6 +5,14 @@ manala_influxdb_install_packages: ~
 manala_influxdb_install_packages_default:
   - influxdb
 
+# Dir
+manala_influxdb_meta_dir: ~
+manala_influxdb_data_dir: ~
+manala_influxdb_wal_dir: ~
+manala_influxdb_data_dir_user: influxdb
```

Comment:
I suggest to rename it to `manala_influxdb_dir_user `, this user is used for data, meta and wal directories creation.